jetty-util/src/main/java/org/eclipse/jetty/util/Utf8Appendable.java

```diff
@@ -251,7 +253,13 @@ protected void appendByte(byte b) throws IOException
                     _codep = 0;
                     _state = UTF8_ACCEPT;
                     _appendable.append(REPLACEMENT);
-                    throw new NotUtf8Exception(reason);
+                    if (b > 0) {
+                        _appendable.append((char)(b & 0xFF));
```

Comment:
This now becomes a security issue (we used to do this, but had to remove it for a vulnerability)